src/dsql/ddl.cpp

```diff
@@ -276,8 +276,9 @@ void DDL_resolve_intl_type(DsqlCompilerScratch* dsqlScratch, dsql_fld* field,
 		return;
 	}
 
-	if (modifying)
+	if (modifying && !field->charSet.hasData() && !field->collate.hasData())
```

Comment:
`! .hasData()` -> `.isEmpty()`

Reply:
May be just `.empty()` then for STL compatibility?